### PR TITLE
Update versions and CHANGELOG.md

### PR DESCRIPTION
**Pull request by OleksiiZubko**
This CR prepares the code for the release.

Resolves: OLPEDGE-2303

Signed-off-by: Oleksii Zubko <ext-oleksii.zubko@here.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+- Added generated functions and interfaces for supporting Authorization APIs. You can find it in file `@here/olp-sdk-dataservice-api/lib/authorization-api-v1.1.ts`.
+
+**olp-sdk-fetch**
+
+- Updated dependencies.
+
+**olp-sdk-dataservice-read**
+
+- Fixed issue with caching partitions metadata without additional fields.
+- Started using RequestFactory from `@here/olp-sdk-core` package
```
Comment (HalynaDumych) [inline CHANGELOG.md:23]:
```suggestion
- Started using `RequestFactory` from the `@here/olp-sdk-core` package.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+
+- Added generated functions and interfaces for supporting Authorization APIs. You can find it in file `@here/olp-sdk-dataservice-api/lib/authorization-api-v1.1.ts`.
+
+**olp-sdk-fetch**
+
+- Updated dependencies.
+
+**olp-sdk-dataservice-read**
+
+- Fixed issue with caching partitions metadata without additional fields.
```
Comment (HalynaDumych) [inline CHANGELOG.md:22]:
```suggestion
- Fixed the issue with caching partitions metadata without additional fields.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,34 @@
...
+- Removed custom header `User-Agent` from the requests.
+
+**olp-sdk-authentication**
+
+- Removed custom header `User-Agent` from the requests.
+- Updated `@here/olp-sdk-fetch` to version 1.6.0.
+
+**olp-sdk-dataservice-api**
+
+- Added generated functions and interfaces for supporting Authorization APIs. You can find it in file `@here/olp-sdk-dataservice-api/lib/authorization-api-v1.1.ts`.
```
Comment (HalynaDumych) [inline CHANGELOG.md:14]:
```suggestion
- Added generated functions and interfaces for supporting Authorization APIs. You can find them in `@here/olp-sdk-dataservice-api/lib/authorization-api-v1.1.ts`.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,34 @@
+## v1.6.0 (21/09/2020)
+
+**olp-sdk-core**
+
+- Removed custom header `User-Agent` from the requests.
+
+**olp-sdk-authentication**
+
+- Removed custom header `User-Agent` from the requests.
```
Comment (HalynaDumych) [inline CHANGELOG.md:9]:
```suggestion
- Removed the `User-Agent` custom header from the requests.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,34 @@
+## v1.6.0 (21/09/2020)
+
+**olp-sdk-core**
+
+- Removed custom header `User-Agent` from the requests.
```
Comment (HalynaDumych) [inline CHANGELOG.md:5]:
```suggestion
- Removed the `User-Agent` custom header from the requests.
```

**Review by HalynaDumych — CHANGES_REQUESTED** [review]

**Review by HalynaDumych — DISMISSED** [review]
LGTM (documentation)

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,34 @@
+## v1.6.0 (21/09/2020)
+
+**olp-sdk-core**
+
+- Removed the `User-Agent` custom header from the requests.
```
Comment (andescu) [inline CHANGELOG.md:5]:
```suggestion
- Removed the `User-Agent` custom header from the requests. This was necessary due to issues sending a custom user-agent in Mozilla browsers.
```

Same for the line below.

**Review by andescu — APPROVED** [review]

**Review by ystefinko — APPROVED** [review]

**Review by andescu — APPROVED** [review]

**Pull request merged**